swap/src/protocol/alice/swap.rs

```diff
@@ -210,8 +210,10 @@ where
                                 bail!("Waiting for Bitcoin transaction finality failed with {}! The redeem transaction was published, but it is not ensured that the transaction was included! You're screwed.", e)
                             }
                         },
-                        Err(e) => {
-                            error!("Publishing the redeem transaction failed with {}, attempting to wait for cancellation now. If you restart the application before the timelock is expired publishing the redeem transaction will be retried.", e);
+                        Err(error) => {
+                            error!(
+                                %error,
+                                "Publishing the redeem transaction failed. Attempting to wait for cancellation now. If you restart the application before the timelock is expired publishing the redeem transaction will be retried.");
```

Comment:
I like how this log message knows about entire program's behaviour :upside_down_face:

Reply:
I think it can be stripped down to `Publishing the redeem transaction failed.` - this is a rather delicate scenario where interaction from an ASB provider might be required that's why that much context was added.
Can we assume that an ASB provider will have to have this knowledge anyway...?

Reply:
Done